gateway/api/localauth/register.go

```diff
@@ -0,0 +1,142 @@
+package localauthapi
+
+import (
+	"fmt"
+	"libhoop/log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/google/uuid"
+	"github.com/hoophq/hoop/gateway/appconfig"
+	"github.com/hoophq/hoop/gateway/pgrest"
+	pglocalauthsession "github.com/hoophq/hoop/gateway/pgrest/localauthsession"
+	pgorgs "github.com/hoophq/hoop/gateway/pgrest/orgs"
+	pgusers "github.com/hoophq/hoop/gateway/pgrest/users"
+	"github.com/hoophq/hoop/gateway/storagev2/types"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// If the system is set as single tenant,
+// we default the new user to the default organization.
+// Otherwise, a new organization is created for the user.
+func manageOrgCreation(user pgrest.User) (string, error) {
+	var tenancy string
+	if pgusers.IsOrgMultiTenant() {
+		tenancy = "multi-tenant"
+	} else {
+		tenancy = "single-tenant"
+	}
+	switch tenancy {
+	case "multi-tenant":
```

Comment:
What's the motivation to add multi tenant to local auth? Our current multi tenant cloud setup won't be using this.

I'm not against having this, but there's too many things to test it in a single PR.
Did you test this setup running the gateway in multi tenant mode?

Reply:
that was added at another moment, on another comprehension of the feature. I just removed it